packages/slate/src/changes/with-schema.js

```diff
@@ -67,42 +66,26 @@ function normalizeNodeAndChildren(change, node, schema) {
     normalizeNode(change, node, schema)
     return
   }
+  const checkedKeys = []
+  let child = node.nodes.first()
+
+  let nodePath = change.value.document.getPath(node.key)
 
   // We can't just loop the children and normalize them, because in the process
   // of normalizing one child, we might end up creating another. Instead, we
   // have to normalize one at a time, and check for new children along the way.
   // PERF: use a mutable array here instead of an immutable stack.
```

Comment:
I think this last line with the `PERF:` comment might be outdated and we can remove it?

Reply:
Yes, I will delete it.